cli/cmd/pro.go

```diff
@@ -232,6 +233,74 @@ var centralInstallCmd = &cobra.Command{
 	},
 }
 
+var migrateCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrate Odigos from Community to Enterprise tier",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		client := cmdcontext.KubeClientFromContextOrExit(ctx)
+
+		onPremToken := cmd.Flag("onprem-token").Value.String()
+		if onPremToken == "" {
+			fmt.Println("\033[31mERROR\033[0m onprem-token is required")
+			os.Exit(1)
+		}
+
+		ns, err := resources.GetOdigosNamespace(client, ctx)
+		if resources.IsErrNoOdigosNamespaceFound(err) {
+			fmt.Println("\033[31mERROR\033[0m no odigos installation found in the current cluster")
+			os.Exit(1)
+		} else if err != nil {
+			fmt.Printf("\033[31mERROR\033[0m Failed to check if Odigos is already installed: %s\n", err)
+			os.Exit(1)
+		}
+
+		fmt.Println("Starting migration from Community to Enterprise tier...")
+
+		odigosConfig, err := resources.GetCurrentConfig(ctx, client, ns)
+		if err != nil {
+			fmt.Printf("Error getting config maps: %v\n", err)
+			return
```

Comment:
nit: for consistency with other checks here

```suggestion
			os.Exit(1)
```